### PR TITLE
Adding port of invoice_voucher_lines_fund_distributions derived table to Metadb

### LINKED ISSUE #532: RM - invoice_voucher_lines_fund_distributions to Metadb

### PR DESCRIPTION
**Pull request by adoerrer**
will fix #532

### CODE REVIEW

**Review by jenmawe — COMMENTED** [review]
I noticed in some of your other derived tables that where you have a number you cast it as ::int. Did you want to do that here for voucher number and invoice voucher lines external account number?

**Comment by scotttperry** [comment]
varchar in both cases sounds right to me. There is a prefix option on the voucher number that can be any character(s).

**Review by lib-eric — APPROVED** [review]
I am unfamiliar how vouchers work so I do not know about the logic there but other then that things look good to me.

- [x] Query runs
- [x] Data output makes sense
- [x] Data types look correct
- [x] Query logic seems correct
- [x] Column names make sense and are spelled correctly
- [x] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)

For derived tables:
- [x] query has indexes on columns
- [x] vacuum analyze has been added to the bottom
- [x] table appears in runlist.txt at an appropriate point

**Review by jenmawe — APPROVED** [review]
Like Eric, I'm unfamiliar with this. Everything worked and I checked all the points and it looks good.

- [x] Query runs
- [x] Data output makes sense
- [x] Data types look correct
- [x] Query logic seems correct
- [x] Column names make sense and are spelled correctly
- [x] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)

For derived tables:
- [x] query has indexes on columns
- [x] vacuum analyze has been added to the bottom
- [x] table appears in runlist.txt at an appropriate point

File: sql_metadb/derived_tables/invoice_voucher_lines_fund_distributions.sql [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        jsonb_extract_path_text(invvl.jsonb, 'amount')::numeric(19,4) AS invoice_voucher_lines_amount,
+        jsonb_extract_path_text(dist.data, 'distributionType') AS fund_distribution_type,
+        jsonb_extract_path_text(invvl.jsonb, 'externalAccountNumber') AS invoice_voucher_lines_external_account_number,
+        voucherid AS voucher_id
+    FROM
+        folio_invoice.voucher_lines AS invvl
+        CROSS JOIN jsonb_array_elements(jsonb_extract_path(invvl.jsonb, 'fundDistributions')) AS dist(data)
+)
+SELECT
+	invoice_voucher_line_id AS invoice_voucher_line_id,
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/invoice_voucher_lines_fund_distributions.sql:22]: Some formatting adjustments are needed such as this tab character.  I will make the changes.

**Pull request merged**